scripts/shared/deploy.sh

```diff
@@ -67,6 +69,11 @@ function create_catalog_source() {
   echo "[INFO](${cluster}) Catalog source ${cs} created"
 }
 
+function schedule_dummy_pod_on_all_clusters() {
+    run_subm_clusters create_namespace ${KINDNET_WORKAROUND_NS}
+    run_subm_clusters deploy_daemonset ${KINDNET_WORKAROUND_NS} "${RESOURCES_DIR}"/dummypod.yaml
```

Comment:
Should we delete the daemonset/namespace?

Reply:
The daemonSet pod runs the command "sleep infinity". They will be deleted when the KIND clusters are deleted.